### PR TITLE
Fix oracle fork test of stETH price from OETH Oracle Router

### PR DESCRIPTION
**Pull request by naddison36**

### CODE REVIEW

File: contracts/test/oracle/oracle.fork-test.js [hunk 1]
```diff
@@ -41,8 +41,8 @@ forkOnlyDescribe("ForkTest: Oracle Routers", function () {
       const { stETH } = fixture;
 
       const price = await oethOracleRouter.price(stETH.address);
-      expect(price).to.approxEqualTolerance(parseUnits("1", 18), 1);
-      expect(price).to.not.eq(parseUnits("1", 18));
+      expect(price).to.lte(parseUnits("1", 18));
```
[the remark this answers is not in the document]
Comment (sparrowDom) [reply contracts/test/oracle/oracle.fork-test.js:44]: This should fail with certain oracle rounds. [Here is](https://etherscan.io/address/0x86392dc19c0b719886221c78ab11eb8cf5c52812#readContract) the chainlink stETH/ETH feed. And if you query roundData for round 18446744073709552382 you will get the price that is above 1: 
Screenshot 2023-09-19 at 16 08 02

**Review by sparrowDom — COMMENTED** [review]
Oracle price for the asset can be above 1

**Comment by naddison36** [comment]
I thought the Oracle price was capped at 1 but that logic is in the vault for redeems and, not the Oracle Router.

I'll fix

File: contracts/test/oracle/oracle.fork-test.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (naddison36) [reply contracts/test/oracle/oracle.fork-test.js:44]: Yes, my bad. I was mistaking the logic for redeems that ensures the price is <= 1.

**Review by sparrowDom — APPROVED** [review]
LGTM

**Pull request merged**